fuzz/fuzz_targets/roundtrip.rs

```diff
@@ -0,0 +1,76 @@
+#![no_main]
+
+use png::FilterType;
+#[macro_use] extern crate libfuzzer_sys;
+extern crate png;
+
+fuzz_target!(|data: &[u8]| {
+    if let Some((raw, encoded)) = encode_png(data) {
+        let (_info, raw_decoded) = decode_png(&encoded);
+        // raw_decoded can be padded with zeroes at the end, not sure if that's correct
+        let raw_decoded = &raw_decoded[..raw.len()];
```

Comment:
Yes, that's correct. The buffer is for a full frame width×height, but with APNG and animation control chunks we might have frames smaller than that initial size.

Reply:
I'm actually seeing the resulting decoded buffer being *larger* than the original input buffer.

That happens only with `image-png` - [`zune-png` always roundtrips the same buffer size](https://github.com/etemesi254/zune-image/pull/22) (although it doesn't handle APNG).